lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -22,7 +22,7 @@
     logger.setLevel(LOG_LEVEL)
 except ValueError:
     logger.setLevel('INFO')
-    logger.warning('Invalid log level specified, defaulting to INFO')
+    logger.error('Invalid log level specified.  Defaulting to INFO.')
```

Comment:
`exception` should be used here to collect the details.

Reply:
done